packages/ui-core/src/components/card/card.component.tsx

```diff
@@ -3,28 +3,30 @@ import { colors } from '@keen.io/colors';
 
 export type Props = {
   backgroundColor?: string;
-  border?: string;
-  borderRadius?: string;
+  borderRadius?: number;
   hasShadow?: boolean;
   hideOverflow?: boolean;
   hasPadding?: boolean;
+  borderWidth?: number;
+  borderColor?: string;
+  padding?: number;
 };
 
 export const Card = styled.div<Props>`
   display: flex;
   flex-direction: column;
   height: 100%;
   background: ${(props) => props.backgroundColor};
-  border: ${(props) => props.border};
-  border-radius: ${(props) => props.borderRadius};
+  border: ${(props) => `${props.borderWidth}px solid ${props.borderColor}`};
+  border-radius: ${(props) => props.borderRadius}px;
   box-sizing: border-box;
   box-shadow: ${(props) =>
     props.hasShadow ? '0px 2px 4px 0px rgba(29,39,41,0.15)' : 'none'};
 
   ${(props) =>
     props.hasPadding &&
```

Comment:
Maybe we could get rid of `hasPadding` property in favor of just passing padding value. 
Then just set padding to `0` in this two places ? What do You think ?
<img width="570" alt="Zrzut ekranu 2021-05-24 o 08 50 48" src="https://user-images.githubusercontent.com/7753369/119308323-30232980-bc6d-11eb-83ec-02994b8b98a1.png">